new-dti-website/components/team/MemberGroup.tsx

```diff
@@ -0,0 +1,352 @@
+                        alt={icon}
+                        height={iconDetails.height}
+                        width={iconDetails.width}
+                      />
+                    </a>
+                  )
+                );
+              })}
+            </div>
+          </div>
+          <div>
+            <button
+              onMouseEnter={() => setHover(true)}
+              onMouseLeave={() => setHover(false)}
+              className="py-3 px-5 bg-white rounded-xl text-[#A52424] border-[3px] border-[#A52424] 
+              hover:bg-[#A52424] hover:text-white"
+            >
+              <div className="flex gap-3 ">
+                <Image
+                  src="/icons/calendar.svg"
+                  alt="calendar"
+                  width={24}
+                  height={24}
+                  className={hover ? 'brightness-0 invert' : ''}
+                />
+                <p className="font-bold text-lg text-inherit">Chat with me</p>
+              </div>
+            </button>
+          </div>
+        </div>
+      </div>
+      <div>
+        <Image
+          src="/icons/close.svg"
+          width={23}
+          height={23}
+          alt="close"
+          onClick={props.onClose}
+          className="m-3 cursor-pointer"
+        />
+      </div>
+    </Card>
+  );
+};
+
+type MemberGroupProps = {
+  roleName: string;
+  description: string;
+  members: IdolMember[];
+  setSelectedMember: Dispatch<SetStateAction<IdolMember | undefined>>;
+  selectedMember: IdolMember | undefined;
+  selectedRole: string;
+  memberDetailsRef: RefObject<HTMLInputElement>;
+};
+
+const MemberGroup: React.FC<MemberGroupProps> = ({
+  roleName,
+  description,
+  members,
+  setSelectedMember,
+  selectedMember,
+  selectedRole,
+  memberDetailsRef
+}) => {
+  const selectedMemberIndex: number = useMemo(
+    () => (selectedMember ? members.indexOf(selectedMember) : -1),
+    [members, selectedMember]
+  );
+
+  const canInsertMemberDetails = (index: number, columns: number): boolean =>
+    selectedMember !== undefined &&
+    (roleName === selectedRole || selectedRole === 'Full Team') &&
+    ((index % columns === columns - 1 &&
+      selectedMemberIndex >= index - columns + 1 &&
+      selectedMemberIndex <= index) ||
+      (index === members.length - 1 &&
+        selectedMemberIndex >= members.length - (members.length % columns)));
+
+  return (
+    (selectedRole === roleName || selectedRole === 'Full Team') && (
+      <div className="mb-[120px]">
+        <h2 className="font-semibold text-[32px]">{`${roleName} ${
+          roleName !== 'Leads' ? '' : 'Team'
+        }`}</h2>
+        <p className="mt-3 text-xl">{description}</p>
+        <div className="flex flex-row justify-between flex-wrap grid lg:grid-cols-4 md:grid-cols-3 gap-10 mt-10 mb-[100px]">
+          {members.map((member, index) => (
+            <>
+              <MemberCard
+                {...member}
+                image="martha.png"
```

Comment:
This is a fairly small thing but I'm guessing that `martha.png` was just pulled off Google since the image dimensions aren't very nicely rounded. It probably wouldn't hurt to at least crop the image to whatever the expected size of headshots should be according to the design. 

(I just say this because knowing how projects go, this will probably be forgotten about til the actual headshots get put in after this PR gets merged). 

This repo has an `image-cli` that should be able to crop (and also compress but that isn't particularly important rn) to the desired size. @oscarwang20 should be aware of it if not @andrew032011 definitely should be. 

Now if Martha would just fix her weather machine...